docs/lib/datastore/fragments/ios/getting-started/40_codegen.md

```diff
@@ -0,0 +1,20 @@
+The `Amplify/Tools` pod can be used to generate the models from the `schema.graphql` and automatically add the new files to the project.
```

Comment:
Suggestion:
If you have chosen you use the `Amplify/Tools` pod, we will now use it to generate the models from the schema defined in `schema.graphql` and automatically add the new files to the project.  If you have chosen to use the Amplify CLI instead, please skip this section, and follow the "Code generation: Amplify CLI" section.

Reply:
cool, loved it